build/images/scripts/install_cni

```diff
@@ -2,25 +2,38 @@
 
 set -euo pipefail
 
+# Fetching the list of the binaries that user wants to skip installing.
+IFS=',' read -r -a binaries <<< "$SKIP_CNI_BINARIES"
 # Todo: check version and continue installation only for a newer version
 
 # Install Antrea binary file
 install -m 755 /usr/local/bin/antrea-cni /host/opt/cni/bin/antrea
 
+# Checking the condition before installing the binaries, to ensure that user
+# does not want to skip the corressponding CNI binary file
+
 # Install the loopback plugin.
 # It is required by kubelet on Linux when using docker as the container runtime.
 # We replace the binary files even they are already present on the Node to make
 # sure expected versions are used.
-install -m 755 /opt/cni/bin/loopback /host/opt/cni/bin/loopback
+if [[ ! " ${binaries[*]} " =~ " loopback " ]]; then
```

Comment:
curious as to why we leave spaces on both sides of the strings?

Reply:
Saw the reason there: https://github.com/antrea-io/antrea/pull/3454/files/3a0ef435fe4524b1527d7e09eece0d19d083a289#diff-7c49c01432e7ddc0627cb150c9d01228d28d49a99cc93573795bc6633cc9c810